web/services/hosts_next.go

```diff
@@ -40,14 +49,23 @@ func (s *hostsNextService) GetAll(filters map[string][]string) (models.HostList,
 		}
 	}
 
-	err := db.Find(&hosts).Error
+	err := db.Find(&hosts).Order("name").Error
 	if err != nil {
 		return nil, err
 	}
 
 	var hostList models.HostList
-	for _, host := range hosts {
-		hostList = append(hostList, host.ToModel())
+	for _, h := range hosts {
+		host := h.ToModel()
+		host.Health = computeHealth(&h)
+
+		health, ok := filters["health"]
+		if ok && len(health) > 0 {
+			if !internal.Contains(health, host.Health) {
+				continue
+			}
+		}
+		hostList = append(hostList, host)
```

Comment:
Is this idiomatic? I'd do the contrary, if internal contains then append. But it's just personal taste when it comes to `continue`, `break` and stuff like that

Reply:
(Also, could we combine these conditions with an `&&`?)

Reply:
well it is idiomatic but what you proposed might be better